server/neptune/temporalworker/server.go

```diff
@@ -191,3 +202,46 @@ func Healthz(w http.ResponseWriter, _ *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
 	w.Write(data) // nolint: errcheck
 }
+
+func createJobsController(config *neptune.Config, jobOutputHandler *job.OutputHandler) *controllers.JobsController {
+	projectJobsScope := config.Scope.SubScope("getprojectjobs")
+	wsMux := websocket.NewInstrumentedMultiplexor(
+		websocket.NewMultiplexor(
+			config.CtxLogger,
+			controllers.JobIDKeyGenerator{},
+			jobOutputHandler,
+		),
+		projectJobsScope,
+	)
+
+	return &controllers.JobsController{
+		AtlantisVersion:     config.ServerCfg.Version,
+		AtlantisURL:         config.ServerCfg.URL,
+		KeyGenerator:        controllers.JobIDKeyGenerator{},
+		StatsScope:          config.Scope,
+		Logger:              config.CtxLogger,
+		ProjectJobsTemplate: templates.ProjectJobsTemplate,
+		WsMux:               wsMux,
+	}
+}
+
+func createJobOutputHandler(config *neptune.Config) (*job.OutputHandler, error) {
+	storageBackend, err := job.NewStorageBackend(config.LogStreamingJobCfg, config.CtxLogger)
+	if err != nil {
+		return nil, errors.Wrapf(err, "initializing storage backend")
+	}
+
+	jobStore := job.NewStore(storageBackend)
+	logFilter := filter.LogFilter{
+		Regexes: config.TerraformCfg.LogFilters.Regexes,
+	}
+
+	jobOutputChan := make(chan *job.OutputLine)
+	return &job.OutputHandler{
+		JobOutput:        jobOutputChan,
+		JobStore:         jobStore,
+		ReceiverRegistry: jobs.NewReceiverRegistry(),
+		LogFilter:        logFilter,
+		Logger:           config.CtxLogger,
+	}, nil
```

Comment:
Can this not happen as part of a `NewOutputHandler` method in the relevant package?